portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContentExportImport.testcase

```diff
@@ -107,6 +107,97 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-114230. Verify that can import image global structures between instances."
+	@priority = "3"
+	test CanImportExportedImageGlobalStructuresBetweenInstances {
+		property test.name.skip.portal.instance = "WebContentExportImport#CanImportExportedImageGlobalStructuresBetweenInstances";
+
+		task ("Given: new web content with WC structure with an image field is exported") {
+			DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "global");
+
+			DMDocument.addCP(
+				dmDocumentDescription = "DM Document Description",
+				dmDocumentFile = "Document_1.png",
+				dmDocumentTitle = "Document Title");
+
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "global");
+
+			NavItem.gotoStructures();
+
+			WebContentStructures.addCP(
+				structureDescription = "WC Structure Description",
+				structureName = "WC Structure Name");
+
+			DataEngine.addField(
+				fieldFieldLabel = "Image",
+				fieldName = "Image");
+
+			WebContentStructures.saveCP(structureName = "WC Structure Name");
+
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "global");
+
+			NavItem.gotoTemplates();
+
+			WebContentTemplates.addCP(
+				structureName = "WC Structure Name",
+				templateDescription = "WC Template Description",
+				templateFieldNames = "Image",
+				templateName = "WC Template Name");
+
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "global");
+
+			WebContentNavigator.gotoAddWithStructureCP(structureName = "WC Structure Name");
+
+			WebContent.addWithStructureCP(
+				imageFileName = "Document Title",
+				navTab = "Documents and Media",
+				structureName = "WC Structure Name",
+				webContentImage = "Document Title",
+				webContentTitle = "WC Title");
+
+			PortletEntry.publish();
+
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "global");
+
+			LAR.exportPortlet(
+				contentLastRange = "Last",
+				larFileName = "WebContent_portlet_structures.lar");
+		}
+
+		task ("When: The exported data is imported into a new virtual instance site") {
+			ApplicationsMenu.gotoPortlet(
```

Comment:
Please use 
HeadlessPortalInstanceAPI.addPortalInstance(
				domain = "www.able.com",
				portalInstanceId = "www.able.com",
				virtualHost = "www.able.com");
to replace line168-176.